Add pushy@1.0.0 w/ git auto-update config

x09326 opened the pull request:
PR for #8301

@pvnr0082t 
Please help me review it, thanks.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib
- Git repository (required):https://github.com/christophery/pushy
- Official website (optional, not the repository):https://www.christopheryee.ca/pushy/
- NPM package url (optional):
- GitHub / Bitbucket popularity (required):
  - Count of stars:1154
  - Count of watchers:61
  - Count of forks:191
- NPM download stats (optional):
  - Downloads in the last day:
  - Downloads in the last week:
  - Downloads in the last month:
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR
- [x] The lib has notable popularity
  - [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [ ] More than 500 downloads stats per month on npm registry
- [x] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

close #8301, cc @christophery

pvnr0082t commented on ajax/libs/pushy/package.json:
```diff
@@ -0,0 +1,28 @@
+    "basePath": "",
+    "files": [
+      "css/pushy*",
+      "js/pushy*"
```
You can use `"+(css|js)/pushy*"` here.

PeterDaveHello replied:
@x09326 @pvnr0082t ...

pvnr0082t replied:
oh...I see now.

pvnr0082t commented on ajax/libs/pushy/1.0.0/css/pushy.css:
```diff
@@ -0,0 +1,172 @@
+  position: absolute;
+  top: 50%;
+  right: 15px;
+  background: url("../img/arrow.svg") no-repeat;
```
@x09326 I think file in `img` folder also need to be added.

x09326 commented:
@pvnr0082t 
I have modified the file map in **package.json**.
And I have added img file and minified ```pushy.css```.
Please help me review it, thanks.

pvnr0082t commented on ajax/libs/pushy/package.json:
```diff
@@ -0,0 +1,28 @@
+    "basePath": "",
+    "files": [
+      "+(css|js)/pushy*",
+      "img/arrow.svg"
```
@x09326 I think "img/*.+(svg|gif|png)" will be more flexible.

PeterDaveHello replied:
If you'll like to use this schem, please also add `jpg` extension it that. Personally I prefer use a more flexible for img like `img/**/*`, images are usually not so messy as js/css files do.

x09326 replied:
I have modified it.

x09326 commented:
@pvnr0082t 
done!

pvnr0082t commented:
LGTM

PeterDaveHello commented:
@x09326 I suggest you to reply on the place you got the comment/request for change.

x09326 commented:
@pvnr0082t 
I have modified the file map to ```img/**/*```.
Please help me review it, thanks.

pvnr0082t commented:
LGTM

PeterDaveHello commented:
@x09326 @pvnr0082t as you can see if you don't reply on the origin review place you'll miss it.

x09326 commented:
ping @pvnr0082t @PeterDaveHello 
I have rebased with the latest master br.
Please help me review it, thanks.

x09326 commented:
@pvnr0082t  @PeterDaveHello 
The ci test has finished.
Please help me review it, thanks.

kennynaoh commented:
LGTM
